views/season.pug

```diff
@@ -0,0 +1,18 @@
+html
+  head
+    link(rel="stylesheet" href="/css/styles.css")
+  body
+    h1 <a href="..">Kim's Convenience</a>
```

Comment:
`<a href="/">` is the correct way of doing this. `..` will cause issues in production.